api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/components/operations/LoopOperation.java

```diff
@@ -0,0 +1,221 @@
+package edu.cornell.mannlib.vitro.webapp.dynapi.components.operations;
+
+import static java.lang.String.format;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import edu.cornell.mannlib.vitro.webapp.dynapi.components.NullProcedure;
+import edu.cornell.mannlib.vitro.webapp.dynapi.components.OperationResult;
+import edu.cornell.mannlib.vitro.webapp.dynapi.components.Parameter;
+import edu.cornell.mannlib.vitro.webapp.dynapi.components.Parameters;
+import edu.cornell.mannlib.vitro.webapp.dynapi.components.Procedure;
+import edu.cornell.mannlib.vitro.webapp.dynapi.components.ProcedureDescriptor;
+import edu.cornell.mannlib.vitro.webapp.dynapi.data.DataStore;
+import edu.cornell.mannlib.vitro.webapp.utils.configuration.Property;
+
+public class LoopOperation extends AbstractOperation {
```

Comment:
We are using LoopOperation for loops, we don't have LoopStep, right?

Reply:
Yes.